tests/testthat/test_ginspectfile_csvgz.R

```diff
@@ -0,0 +1,58 @@
+library(GGIR)
+testthat::context("g.inspectfile CSV")
+
+# adapted from R.utils::compressFile
```

Comment:
R.utils uses LGPL license, by which including their code is a license violation.

Proposal:
Replace the quick_gzip function you added to the test by

```
quick_gzip = function(filename) {
  fext = strsplit(filename, "[.]")[[1]]
  fext = fext[length(fext)]
  destnameT = tempfile(fileext = paste0(".", fext, ".gz"))
  x = read.csv(file = filename)
  data.table::fwrite(x = x, file = destnameT, compress = "gzip")
  return(destnameT)
}
```

The test still passes and it uses the already existing data.table dependency.

Reply:
I'm going to do this since the header is really weird (it's actigraph header) so just reading/writing may be more general.
```r
quick_gzip = function(filename) {
  fext = strsplit(filename, "[.]")[[1]]
  fext = fext[length(fext)]
  destnameT = tempfile(fileext = paste0(".", fext, ".gz"))
  x = readLines(filename, warn = FALSE)
  writeLines(x, destnameT)
  return(destnameT)
}
```

Reply:
```
x = readLines(filename, warn = FALSE)
writeLines(x, destnameT)
```
this does not actually gzip the file but saves the file with the csv.gz extension? If that still offer a realistic test of the functionality then in that case we could as well just rename it with `file.rename(fext, destnameT)` and pretend the file was gunzipped.

Reply:
I've reworked this so it actually gzips the data.